### DISCUSSION TITLE
Termux isn't work

### DISCUSSION THREAD

**Discussion by Variable-py** [community/community#29713] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi ,
When I open Termux in my phone it's say Termux unable to install bootstrap pakages

**Comment by queenofcorgis** [comment, marked as answer]
If you're asking how to use the piece of software that you found in a repository on GitHub, the best way to contact the maintainers of that software is to:
* Check the README for instructions on how to operate the software and pointers to documentation or troubleshooting info.
* Check the [SUPPORT file](https://help.github.com/articles/adding-support-resources-to-your-project/), if one exists, for instructions on how to best contact the maintainers for support.
* Check the [CONTRIBUTING guide](https://help.github.com/articles/setting-guidelines-for-repository-contributors/), if one exists. Sometimes if there isn't a SUPPORT file, the CONTRIBUTING guide will give instructions on how to contact the maintainers for support.

All of these documents can be found, if they exist, in the repository where you found the software itself.
